packages/jest-worker/__benchmarks__/test.js

```diff
@@ -5,22 +5,22 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+// Typical tests: node --expose-gc test.js empty 100000
+//                node --expose-gc test.js loadTest 10000
+
 'use strict';
 
 const assert = require('assert');
 const {performance} = require('perf_hooks');
-// eslint-disable-next-line import/no-extraneous-dependencies
 const workerFarm = require('worker-farm');
-const JestWorker = require('../../build').Worker;
+const JestWorker = require('../build').Worker;
 
-// Typical tests: node --expose-gc test.js empty 100000
-//                node --expose-gc test.js loadTest 10000
```

Comment:
should keep this

Reply:
Just moved it up for visibility. Also I added similar instructions to other scripts.